### PR TITLE
Adds the Volume of a Gas Tank to its Examine Text

### PR DESCRIPTION
**Pull request by Cojoke-dot**
## About the PR

Allows players to view the volume of a gas tank through its examine text.
## Why / Balance

Helps clarify the size difference between tanks more easily
## Technical details

Just a PushMarkup 
## Media

Screenshot 2024-07-06 210230Screenshot 2024-07-06 210222

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: You can now read the volume of a gas tank in its examine text

### CODE REVIEW

**Comment by Partmedia** [comment]
Thank you for this PR. Conceptually, I think this is fine. However, we already merged #25720, where many of the reasons for doing so would sort of be superseded by this examine text being available. Can you discuss a bit more about why this is needed, and whether or not the volume reading should be removed from the gas analyzer?

**Comment by deltanedas** [comment]
only people analyzing a tank are:
- people making bombs
- people making cmb internals
- metagamers checking if their oxygen tank from engi is full of plasma

this will help for people that just want to know how good a random tank is for breathing

**Comment by Cojoke-dot** [comment]
> only people analyzing a tank are:
> 
> * people making bombs
> * people making cmb internals
> * metagamers checking if their oxygen tank from engi is full of plasma
> 
> this will help for people that just want to know how good a random tank is for breathing

yeah, you summed it up well, its a label on the tank for the public to read on the fly without technical equipment.

**Comment by EmoGarbage404** [comment]
from a UX concern, the major issue with this is displaying a variable value (pressure) alongside a constant one (volume). One is a statement of the gas inside and the other is one of the container. Having them both visible side by side in the same way is kinda confusing. I would expect the volume to be correlated in some meaningful way to the pressure value.

**Comment by Cojoke-dot** [comment]
> correlated in some meaningful way to the pressure value.

Where is a better place I could put its to reduce the correlation?

**Comment by Plykiya** [comment]
I tried to breathe out of my shoes once and was confused when it ran out of contents in like 20 seconds

Knowing the volume of a tank gives you some idea as to how long you can expect some gas tank to last over another, the pressure itself is nearly pointless for making comparisons between two different tanks

**Comment by luizwritescode** [comment]
I think the variable value being orange and the constant value being gray is already a good way to differ both. If it's not good enough then I'd suggest adding it directly to the description like:

`A standard cylindrical gas tank for oxygen. It can hold 5L of gas.`

**Comment by slarticodefast** [comment]
I added the volume to the gas analyzer for pretty much the same reason. I guess you are right that a lot of players that do not play atmos rarely use a gas analyzer and don't know the tank size. But since this is a constant value that only has to be looked up once it should be done in a way that does not unnecessarily clutter the interface. Personally I like the above suggestion of putting it in the description or alternatively the name "oxygen tank (5L)".
If we don't have that already we could also add a guidebook entry about internals usage, which could list the different tank sizes and their usage duration.

**Comment by deltanedas** [comment]
a guidebook entry would get outdated faster than a description would if someone changes their volumes

**Comment by Cojoke-dot** [comment]
Yeah, my biggest issue with the description is for future items, same kinda thing as the guidebook. I put it in the examine text so it would go automatically for all gas tanks but I could go through and put them all in manually I guess, people would just have to remember to add the volume in the description for the future.

**Comment by Tayrtahn** [comment]
> a guidebook entry would get outdated faster than a description would if someone changes their volumes

If only there were a way to use the actual values in the guidebook... #27570

**Comment by Cojoke-dot** [comment]
> `A standard cylindrical gas tank for oxygen. It can hold 5L of gas.`

I mad it look like this 👍

File: Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml [hunk 1]
```diff
@@ -206,7 +206,7 @@
   parent: GasTankRoundBase
   id: AirTank
   name: air tank
-  description: Mixed anyone?
+  description: Mixed anyone? It can hold 5L of gas.
```
Comment (Plykiya) [inline Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml:209]: add a space between 5 and L, otherwise good to merge

**Comment by Plykiya** [comment]
I'm still mad about being able to breathe through my shoes

**Comment by JIPDawg** [comment]
> I'm still mad about being able to breathe through my shoes

I thought you ate your shoes.

**Pull request merged**